HeroVault/HeroVault/ServiceLayer/Constants.swift

```diff
@@ -24,5 +24,6 @@ struct Constants {
         static let homeScreenDetailSegueName = "SearchDetailsSegue"
         static let homeScreenTableViewCellName = "HomeScreenTableViewCell"
         static let loginScreenSegueName = "logInSegue"
```

Comment:
The strings must be consistent. Some start with a capital letter like "SearchDetailsSegue" and other start with a lower letter like "logInSegue". Choose a standard and stick with it. I suggest making them all uppercase😄